core/src/main/scala/kafka/coordinator/group/GroupMetadata.scala

```diff
@@ -677,11 +677,17 @@ private[group] class GroupMetadata(val groupId: String, initialState: GroupState
     }
   }
 
-  def activeProducers = pendingTransactionalOffsetCommits.keySet
+  def activeProducers: collection.Set[Long] = pendingTransactionalOffsetCommits.keySet
 
-  def hasPendingOffsetCommitsFromProducer(producerId: Long) =
+  def hasPendingOffsetCommitsFromProducer(producerId: Long): Boolean =
     pendingTransactionalOffsetCommits.contains(producerId)
 
+  def hasPendingOffsetCommitsForTopicPartition(topicPartition: TopicPartition) : Boolean = {
```

Comment:
It is also possible that the coordinator has non-transactional, pending offset commits. I am wondering if we should generalize this patch to handle both cases.